# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: .cargo/config.in
# Commit: 5762b27d2cc8
# Full Hash: 5762b27d2cc82e6c7f3f0d37bc4bcfdb130f9220
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2023-06-21 04:00:08
# Description:
#   Bug 1839459 - Elide empty closed sub-paths instead of emitting a start point. r=jrmuizel
#   
#   There is a corner case that triggers an assert in wpf-gpu-raster if an empty sub-path
#   starts off a path. Just avoid the problem by eliding the sub-path, but at least handle
#   close correctly as per the Canvas2D closePath spec:
# ==============================================================================

diff -r caa96b0874f4 -r 5762b27d2cc8 .cargo/config.in
--- a/.cargo/config.in	Tue Jun 20 18:01:09 2023 +0000
+++ b/.cargo/config.in	Tue Jun 20 18:04:20 2023 +0000
@@ -10,9 +10,9 @@
 rev = "07d3c25322518f294300e96246e09b95e118555d"
 replace-with = "vendored-sources"
 
-[source."git+https://github.com/FirefoxGraphics/wpf-gpu-raster?rev=112a2140c433af06938c663fc6777e2bf17bb6ce"]
+[source."git+https://github.com/FirefoxGraphics/wpf-gpu-raster?rev=5ab6fe33d00021325ee920b3c10526dc8301cf46"]
 git = "https://github.com/FirefoxGraphics/wpf-gpu-raster"
-rev = "112a2140c433af06938c663fc6777e2bf17bb6ce"
+rev = "5ab6fe33d00021325ee920b3c10526dc8301cf46"
 replace-with = "vendored-sources"
 
 [source."git+https://github.com/chris-zen/coremidi.git?rev=fc68464b5445caf111e41f643a2e69ccce0b4f83"]